modules/xml-sitemap/sitemap.php

```diff
@@ -0,0 +1,728 @@
+<?php
+/**
+ * Generate sitemap files in base XML as well as popular namespace extensions.
+ *
+ * @author Automattic
+ * @version 2.0
```

Comment:
Why do we need this version number? Can we drop it to avoid confusion with Jetpack 2.0?